tensorflow_graphics/projects/mesh_rcnn/ops/cubify.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2020 The TensorFlow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Implementation of the cubify operation for Mesh R-CNN."""
+
+import tensorflow as tf
+
+from tensorflow_graphics.projects.mesh_rcnn.structures import mesh
+
+
+def _ravel_index(index, dims):
+  """
+  Computes a linear index in an array of shape dims.
+  Reverse functionality of tf.unravel_index.
+  Args:
+    index: A Tensor with each row corresponding to indices into an array
+    of dimension dims.
+    dims: The shape of the array to be indexed.
+
+  Note:
+    Only supports dims=(H,W,D)
```

Comment:
What axes does H, W, D correspond to? Is it along x, y, z? it will be nice to document what we mean by HWD. Also somewhere else I also see DHW. We should be consistent.